lib/datadog/tracing/span_attribute_schema.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Datadog
+  module Tracing
+    # Contains methods for fetching values according to span attributes schema
+    module SpanAttributeSchema
+      module Ext
+       DEFAULT_VERSION = 'v0'.freeze
+       VERSION_ONE = 'v1'.freeze
+      end
+
+      module_function
+
+      def fetch_service_name(env, default)
+        ENV.fetch(env) do
+          if Datadog.configuration.tracing.span_attribute_schema == Ext::VERSION_ONE
+            Datadog.configuration.service
+          else
+            default
+          end
+        end
+      end
+
+      def default_span_attribute_schema?
+        Datadog.configuration.tracing.span_attribute_schema == Ext::DEFAULT_VERSION
+      end
+
+      def get_schema_version_numeric(version = Ext::DEFAULT_VERSION)
```

Comment:
The pattern in Ruby is to not have `get_` as a prefix, but instead simply have the method name `schema_version_numeric`.
The equivalent `set_` would be `def schema_version_numeric=()`.